internal/store/horizontalpodautoscaler.go

```diff
@@ -184,6 +184,68 @@ func hpaMetricFamilies(allowAnnotationsList, allowLabelsList []string) []generat
 				return &metric.Family{Metrics: ms}
 			}),
 		),
+		*generator.NewFamilyGenerator(
+			"kube_horizontalpodautoscaler_status_current_metric",
```

Comment:
Maybe this would be a more consistent name:
```suggestion
			"kube_horizontalpodautoscaler_status_target_metric",
```